tests/src/Kernel/Entity/SelectionHandlerTest.php

```diff
@@ -117,8 +117,6 @@ protected function setUp() {
    */
   public function testSelectionHandler() {
     $this->assertEquals(get_class($this->selectionHandler->getSelectionHandler()), 'Drupal\node\Plugin\EntityReferenceSelection\NodeSelection');
-    $this->assertEquals($this->selectionHandler->getConfiguration('handler'), 'default:node');
-    $this->assertEquals($this->selectionHandler->getConfiguration('target_type'), 'node');
```

Comment:
I think we still need this one to see we got the original entity selection handler